fastlane_core/lib/fastlane_core/cert_checker.rb

```diff
@@ -90,8 +90,7 @@ def self.wwdr_certificates_installed?
     end
 
     def self.install_wwdr_certificates
-      install_wwdr_certificate('https://developer.apple.com/certificationauthority/AppleWWDRCA.cer')
```

Comment:
@joshdholtz I think removing this should be OK. According to [Apple Worldwide Developer Relations Intermediate Certificate Expiration](https://developer.apple.com/support/expiration/) nobody should be using this any more and it's set to expire on February 7, 2023 anyway, so not much point in installing it.

Reply:
Yep. Thats the reason i removed it.
This old cert also was source of many resign problems.